.gitleaksignore

```diff
@@ -2,3 +2,4 @@
 
 cd9c0efec38c5d63053dd865e5d4e207c0760d91:docs/guides/Perform_static_analysis.md:generic-api-key:37
 87312c6a627a7b0420956d49187fd15b130df170:src/__tests__/components/molecules/LoginStatus.test.tsx:jwt:23
+37ca9f5670f4cd7d91869845ca27defbe6156bb9:src/__tests__/components/molecules/LoginStatus.test.tsx:jwt:23
```

Comment:
Do we actually need this, the line only has `undefined` data?  Can we clear all these down?

Reply:
Didn't realise that they referred to other branches, that seems like a terrible feature, branches should be isolated from each other!